term.go

```diff
@@ -303,7 +320,12 @@ func (t *Terminal) Write(b []byte) (int, error) {
 }
 
 func (t *Terminal) setupShortcuts() {
-	t.ShortcutHandler.AddShortcut(&desktop.CustomShortcut{KeyName: fyne.KeyV, Modifier: fyne.KeyModifierShift | fyne.KeyModifierControl},
+	mod := fyne.KeyModifierShift | fyne.KeyModifierControl
+	if runtime.GOOS == "darwin" {
```

Comment:
Instead of this if you could use `KeyModiferDefault` which toggles between control and super just as this code does